calpip/Task1/Task 1-2.txt

```diff
@@ -1,24 +1,24 @@
 Dear Sir or Madam,
 
 A couple of months ago I bought a long term membership card to one of 
-your gyms located in Winnipeg downtown area. I have paid a considerable 
-high amount of money expecting the best quality of service in return. 
+your gyms located in Winnipeg downtown area. I have paid a considerably 
+high amount of money, expecting the best quality of service in return. 
```

Comment:
`a considerable high` must be `a considerably high` to be correct, note the `-ly`